scripts/test

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -eu
+
+GIT_COMMIT="${GIT_COMMIT:-latest}"
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+Run load tests.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]
+then
+    if [ "${1:-}" = "--help" ]
+    then
+        usage
+    else
+        docker-compose run --rm gatling-sbt gatling:test || true
```

What is the `|| true` block accomplishing here? I pulled it over from Raster Foundry but don't quite understand its purpose.

---

Someone else may have a better answer, but this is typically a technique to mask a failure from propagating in the command to the left of the `||`. Perhaps there are scenarios where running the Gatling tests returns a non-zero exit status but we don't care?

---

Yeah, that's what's happening here. We have a hardcoded value of getting the response times under a certain amount. Technically the test "fails" if we do not meet that target. However, we don't want the script to exit/stop at that point because we want to still upload the results to S3.